archive/src/main/java/io/orkes/conductor/dao/postgres/archive/PostgresArchiveDAO.java

```diff
@@ -513,9 +550,11 @@ public List<WorkflowModel> getWorkflowFamily(String workflowId, boolean summaryO
                 String wfId = rs.getString("workflow_id");
                 String parentWfId = rs.getString("parent_workflow_id");
                 String status = rs.getString("status");
+                String compWithErr = getCompletedWithError(wfId);
```

Comment:
we cannot load the entire workflow just to find the completed with errors value

this code must be fast, especially in the summaryOnly mode
and getWorkflow probably loads entire json and parses it and creates an object ... which is exactly what we are avoiding in the summaryOnly call

Instead, modify the SELECT for SUMMARY to get the completedWithErrors from DB (json_data column) using a regex.

Reply:
done